### ISSUE TITLE
Update failed EADDRINUSE

### ISSUE THREAD

**Issue by reohjs** [klayveR/xenontrade#11]
Getting this error:

`Error: connect EADDRINUSE 104.31.89.66.80`

**Comment by klayveR** [comment]
I assume this happens when the app is trying to update poe.ninja data?

Might be a problem with my [poe-ninja-api-manager](https://github.com/klayveR/poe-ninja-api-manager) module, it might not close the connection successfully after each API call. I'll look into it.

I doubt the following solves the problem, but it can't hurt to try it out: https://stackoverflow.com/a/16432487

**Comment by klayveR** [comment]
Is this still an issue?

**Comment by reohjs** [comment]
Yeah, still an issue with latest version, even after running `taskkill`

**Comment by hadyfayed** [comment]
Hello guys, i like to add a note to this issue it seems that https://poe.ninja don't resolve most of the time for some ppl for me it resolves 1 in 30 try and its not caused by poe.ninja it self but by CloudFlare as poe.ninja use it as a cdn and dns, for me i can access poe.ninja by bypassing cloudflare using there ip address http://94.1#0.1#6.219/ which by the way not reliable as it can get changed and also their http server is configured for the domain only so you cant get any data from there.
i think poe.ninja have to be notified about that so they can investigate it.
ps. i have encrypted the ip for thier privacy and also because i got it in no so legit way
